python/housinginsights/ingestion/LoadData.py

```diff
@@ -312,6 +313,7 @@ def _create_zone_facts_table(self):
         # create empty zone_facts table
         metadata = MetaData(bind=self.engine)
         zone_facts = Table('zone_facts', metadata,
+                           Column('zone_type', String),
```

Comment:
Will do but I'm a little confused. I thought one of the reasons we transitioned to Flask SQLAlchemy ORM was to replace meta.json by implementing models. I'm still playing catch up with all the work you and Jason have done but wanted to make sure I still understand what the end goal looks like. Thanks.